src/main/java/servise/Service.java

```diff
@@ -0,0 +1,35 @@
+package servise;
+
+import entities.Hotel;
+import entities.Room;
+import entities.User;
+
+import java.util.List;
+import java.util.Map;
+
+public class Service {
```

Comment:
1. Call this class more specific - the same as controller.
2. Inject DAO classes